polaris-patterns/src/components/ResourceDetailsLayout/ResourceDetailsLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import type {PropsWithChildren} from 'react';
+import React from 'react';
+import {VerticalStack, HorizontalGrid, AlphaCard} from '@shopify/polaris';
+
+export type ResourceDetailsLayoutProps = PropsWithChildren;
+
+export const ResourceDetailsLayout: React.FunctionComponent<PropsWithChildren> & {
+  PrimarySection: typeof PrimarySection;
+  SecondarySection: typeof SecondarySection;
+  Card: typeof Card;
+} = function ResourceDetailsLayout({children}: PropsWithChildren) {
+  return (
+    <HorizontalGrid columns={{xs: 1, md: '2fr 1fr'}} gap="4">
+      {children}
+    </HorizontalGrid>
+  );
+};
```

Comment:
Should we change anything about these columns as per the [discussion here](https://github.com/Shopify/polaris/pull/7391#issuecomment-1279463546)?